schema/geometry_schema.json

```diff
@@ -444,6 +485,31 @@
 							"vertex_id"
 						]
 					}
+				},
+				"below_floor_plenum_height": {
+					"description": "If provided, overrides the story below_floor_plenum_height.",
+					"type": "number"
+				},
+				"floor_to_ceiling_height": {
+					"description": "If provided, overrides the story floor_to_ceiling_height.",
+					"type": "number",
+					"minimum": 0,
+					"exclusiveMinimum": true
+				},
+				"above_ceiling_plenum_height": {
+					"description": "If provided, overrides the story above_ceiling_plenum_height.",
+					"type": "number"
+				},
+				"floor_offset": {
+					"description": "If provided, specifies the vertical offset between the space floor and the story floor.",
+					"type": "number"
+				},
+				"open_to_below": {
+					"description": "If true, defines a space that is open to below (has no floor).",
```

Comment:
Will we need to do anything fancy with below floor plenums on this story and above ceiling floor plenums on story below if this is true?  Probably can wait until later.